### PR TITLE
feat: add rtl mode support

### PR DESCRIPTION
**Pull request by chsdwn**

### CODE REVIEW

**Comment by frodriguez-hu** [comment]
Hey, for what is this exactly?

**Comment by chsdwn** [comment]
> Hey, for what is this exactly?

In LTR mode dots move from left to right but in RTL mode it needs to be move from right to left. `flexDirection: rtlMode ? 'row-reverse' : 'row'` this line reverses `ScrollView`'s container horizontal direction on RTL mode.

**Comment by frodriguez-hu** [comment]
Could you provide some videos of evidence of how this works? And a description explaining the feature. I am `felire` the owner of the repo but from another account

**Comment by chsdwn** [comment]
You can check before and after videos below:

- Before

https://github.com/felire/react-native-animated-dots-carousel/assets/13810855/23a68bf3-b667-4437-a43e-61c249142c40

- After

https://github.com/felire/react-native-animated-dots-carousel/assets/13810855/31962815-3015-4532-b03d-d38a2481501d

File: README.md [hunk 1]
```diff
@@ -159,7 +160,8 @@ export interface DecreasingDot {
...
 | `decreasingDots`                  | `DecreasingDot[]`                   | required                        | This is a list where you have to define the quantiy per element and the dot config. The quantity represents how many dots with this config you want per side (simetrically). The size of this elements should be decreasing size if you want this to look nice.      
 | `verticalOrientation`                  | `boolean`                   | false                        | If you want this oriented vertically or horizontally. Default is horizontally       
-| `interpolateOpacityAndColor`                  | `boolean`                   | true                        | Default is true. With this setted to true you will be able to see an animation everytime the activeDot change.            
+| `interpolateOpacityAndColor`                  | `boolean`                   | true                        | Default is true. With this setted to true you will be able to see an animation everytime the activeDot change.
+| `rtlMode`                  | `boolean`                   | false                        | Right-to-left mode
```
Comment (felire) [inline README.md]: I do not find this necessary if it will be delegated to the I18nManager

**Comment by felire** [comment]
Hey, thanks for the videos and description, I just left a comment

**Review by felire — APPROVED** [review]

**Pull request merged**